pkg/upgrade/job/job.go

```diff
@@ -233,6 +233,9 @@ func New(plan *upgradeapiv1.Plan, nodeName, controllerName string) *batchv1.Job
 			//only available in kubectl version 1.18 or later
 			args = append(args, "--disable-eviction=true")
 		}
+		if drain.SkipWaitForDeleteTimeout > 0 {
```

Comment:
This should carry the same comment as for `DisableEviction` above, i.e. v1.18 or greater

Reply:
okay, should it be include that in the README as well?